### PR TITLE
1090-enhance-tutorials

### LINKED ISSUE #1090: Tutorials bugs to improve
**Describe the bug**
There are some bugs found when tried to run these tutorials below.
- [ ] Reconstruction tutorial:
    - [ ] some typo and grammar problems in the readme file. 
    - [ ] `fastmri_ssim.py` absents in `varnet_demo` folder.
    - [ ] the checkpoint device directly map to 'cpu' in [unet_demo/inference.ipynb](https://github.com/binliunls/tutorials/blob/1014-mlflow-handler-tutorial/reconstruction/MRI_reconstruction/unet_demo/inference.ipynb).
   

- [ ] Detection tutorial:
     - [ ] some typo, link and grammar problems in the readme file. 
  
- [ ] CAI tutorial:
    - [ ] add explanation for opencv codec error in the `video_seg` tutorial.
    - [ ] use `EnsureChannelFirst` instead of `AsChannelFirst` in `endoscopic inbody classification tutorial`.
    - [ ] accuracy in endoscopic inbody classfication tutorial should be a number instead of tensor.
   
- [ ] Pathology tutorial:
    - [ ] some useless cell output in nuclick tutorial.

- [ ] DeepAtlas tutorial:
    - [ ] the print output run by myself is different from the original one in github.

### PR DESCRIPTION
**Pull request by binliunls**
Signed-off-by: binliu <binliu@nvidia.com>

Fixes #1090  .

### Description
Fix some basic problems occured during test.

### Checks
- [x] Reconstruction tutorial:
    - [x] some typo and grammar problems in the readme file. 
    - [x] `fastmri_ssim.py` absents in `varnet_demo` folder.
    - [x] the checkpoint device directly map to 'cpu' in [unet_demo/inference.ipynb](https://github.com/binliunls/tutorials/blob/1014-mlflow-handler-tutorial/reconstruction/MRI_reconstruction/unet_demo/inference.ipynb).
   

- [x] Detection tutorial:
     - [x] some typo, link and grammar problems in the readme file. 
  
- [x] CAI tutorial:
    - [x] add explanation for opencv codec error in the `video_seg` tutorial.
    - [x] use `EnsureChannelFirst` instead of `AsChannelFirst` in `endoscopic inbody classification tutorial`.
    - [x] accuracy in endoscopic inbody classfication tutorial should be a number instead of tensor.
   
- [x] Pathology tutorial:
    - [x] some useless cell output in nuclick tutorial.
 
- [x] Add license to above tutorials
- [x] Check license in other tutorials

### CODE REVIEW

**Comment by mingxin-zheng** [comment]
Hi @binliunls . Thank you for the PR and making changes on so many files. Below are my comments.

For `paired_lung_ct.ipynb`:
- I suggest to re-run the notebook to get rid of the warning message
    ```
    UserWarning: This DataLoader will create 4 worker processes in total. Our suggested max number of worker in current system is 2,
    ```
    ```
    /usr/local/lib/python3.7/dist-packages/monai/networks/blocks/warp.py:69: UserWarning: monai.networks.blocks.Warp: Using PyTorch native grid_sample.
    warnings.warn("monai.networks.blocks.Warp: Using PyTorch native grid_sample.")
    ```
- The last markdown should be `## Cleanup data directory`
- There is a difference between toolkit and tutorial on the `extract_level` of the `LocalNet`

**Comment by binliunls** [comment]
> * There is a difference between toolkit and tutorial on the `extract_level` of the `LocalNet`

Hi @wyli,
    I saw you updat this `extract_level` parameter in the last commit of the paired_lung_ct tutorial. I think we need to update the toolkit with this change. Could you please help me to double check that?

Thanks,
Bin

**Comment by wyli** [comment]
> > * There is a difference between toolkit and tutorial on the `extract_level` of the `LocalNet`
> 
> Hi @wyli, I saw you updat this `extract_level` parameter in the last commit of the paired_lung_ct tutorial. I think we need to update the toolkit with this change. Could you please help me to double check that? cc: @mingxin-zheng

sure, in monai v1.1.0 we released a bug fix (https://github.com/Project-MONAI/MONAI/commit/c8ef7fe1ca3d2fbed135a60d288063d536a58eb6 ), and the fix broke the tutorial (ticket https://github.com/Project-MONAI/tutorials/issues/1056), then I fixed the tutorial with this update (https://github.com/Project-MONAI/tutorials/commit/5d490ea658b85bcc1351fbfaebee85e2f7a63e92).

**Comment by mingxin-zheng** [comment]
Okay thanks @wyli !

**Review by mingxin-zheng — COMMENTED** [review]
In `video_seg.ipynb`, does the `Video` work in general settings? @binliunls 
```python
Video(inferred_vid, embed=True, height=300)
```

**Comment by mingxin-zheng** [comment]
Hi @binliunls , I found that some notebooks did not follow the guideline formats as reported by `build/guidelins` test. 

Particularly, the notebook `deep_atlas_tutorial.ipynb` needs "Setup environment", "Setup imports" and the Table of Content (ToC) needs to be updated carefully without creating dead links. I would recommend removing the ToC because I don't think it is necessary to have that in a notebook"

File: deep_atlas/deep_atlas_tutorial.ipynb [hunk 1]
```diff
@@ -5,6 +5,17 @@
...
+    "&nbsp;&nbsp;&nbsp;&nbsp;http://www.apache.org/licenses/LICENSE-2.0  \n",
+    "Unless required by applicable law or agreed to in writing, software  \n",
+    "distributed under the License is distributed on an \"AS IS\" BASIS,  \n",
+    "WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  \n",
+    "See the License for the specific language governing permissions and  \n",
+    "limitations under the License.\n",
+    "\n",
     "# DeepAtlas\n",
     "\n",
     "This tutorial demonstrates the use of MONAI for simultaneous training of a registration and a segmentation network, following the approach of [DeepAtlas](https://biag.cs.unc.edu/publication/dblp-confmiccai-xu-n-19/).\n",
```
Comment (mingxin-zheng) [inline deep_atlas/deep_atlas_tutorial.ipynb:21]: Some pip installations are not necessary:
```
!python -c "import numpy" || pip install -q numpy
!python -c "import torch" || pip install -q torch
```

**Review by mingxin-zheng — COMMENTED** [review]
”spleen_segmentation_[aim|mlflow]" notebooks lack copyright info. Please also check if there are any missing license header related to the files in this PR in the `copyright` test. @binliunls

**Comment by rijobro** [comment]
Hi, video doesn't appear to be working for me, either. This stackoverflow post has a few ideas on how to fix if anyone wants to give those a go: https://stackoverflow.com/a/68381044/12411867.

File: reconstruction/MRI_reconstruction/varnet_demo/inference.ipynb [hunk 2]
```diff
@@ -19,6 +31,7 @@
   },
   {
    "cell_type": "markdown",
+   "id": "f85aee82",
    "metadata": {},
    "source": [
     "# Import packages"
```
Comment (mingxin-zheng) [inline reconstruction/MRI_reconstruction/varnet_demo/inference.ipynb:37]: I think this notebook lacks the standard cells for "Setup environments" and "Setup imports"

**Comment by mingxin-zheng** [comment]
> Hi, video doesn't appear to be working for me, either. This stackoverflow post has a few ideas on how to fix if anyone wants to give those a go: https://stackoverflow.com/a/68381044/12411867.

Hi @rijobro @binliunls , if it needs some tweaking to get it work, how about removing this code cell, or replace it by other solutions, such as writing a video file?

**Comment by rijobro** [comment]
I'd be grateful if someone wanted to invest some time to get it working in some capacity. The purpose of the notebook is to segment a video, so it's a shame to not then show the segmented video at the end. 

We could save it to file and allow the user to view externally, but this requires more effort from the user, so fewer people will review the video, especially if using Google Colab or jupyter notebook over SSH.

**Comment by rijobro** [comment]
I'm definitely ok to have a separate issue/pr to fix the video playback! Don't think it should be deleted from the source code in this pr though - - it'll mess up the diff to delete something and then add it back in.

**Review by mingxin-zheng — APPROVED** [review]
